apps/console/src/extensions/components/authenticators/email-otp/quick-start.tsx

```diff
@@ -94,7 +94,7 @@ const EmailOTPQuickStart: FunctionComponent<EmailOTPQuickStartPropsInterface> =
                             i18nKey={ "extensions:develop.identityProviders.emailOTP.quickStart.steps.selectEmailOTP" +
                             ".content" }
                         >
-                            Go to <strong>Sign-in Method</strong> tab and click on <strong>Start with default
+                            Go to <strong>Login Flow</strong> tab and click on <strong>Start with default
```

Comment:
How about defining a constant for "Login Flow" and using it throughout the app?

Reply:
This is just fallback text, IMO, better not to invest a lot here.